src/app/globale/spacer.tsx

```diff
@@ -0,0 +1,20 @@
+import clsx from 'clsx';
```

Comment:
Why are you adding this "spacer.tsx" in a "globale" folder?

Reply:
j'ai mit juste pour configurer les space mais j'ai déjà enlevé.